stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Config.kt

```diff
@@ -8,12 +8,14 @@ public data class Config(
     /**
      * Date of channel creation.
      */
+    @Deprecated("Deprecated. Use createdAt instead")
```

Comment:
These annotations don't have to say "Deprecated" inside the message. That will be displayed in the IDE thanks to the `@Deprecated` annotation anyway.

It'd also be useful to add an explicit `level` and a `replaceWith` parameter in the annotation.